lib/documents/schemas/hmrc_contacts.json

```diff
@@ -0,0 +1,138 @@
+{
+  "base_path": "/find-hmrc-contacts",
```

Comment:
Andrew and I discussed a few alternatives, but are happy with this one.
NB, `https://www.gov.uk/contact-hmrc` (one of the contenders) already exists.

Reply:
This being slightly different from the format may throw my automated test, but I'm sure I can work around it

Reply:
@GDSNewt can you expand on this? If there's a pattern I'm quite happy to follow it, but from a quick look there doesn't seem to be a pattern as far as I can tell:

- `"base_path": "/asylum-support-tribunal-decisions"`,`"format": "asylum_support_decision"`
- `"base_path": "/animal-disease-cases-england"`, `"format": "animal_disease_case"`
- `"base_path": "/data-ethics-guidance"`, `"format": "data_ethics_guidance_document"`
- etc

Reply:
Chatted on Slack, Alex is happy there's nothing to change.